FFXIVClientStructs/FFXIV/Client/Game/Object/Treasure.cs

```diff
@@ -6,12 +6,93 @@ namespace FFXIVClientStructs.FFXIV.Client.Game.Object;
 [Inherits<GameObject>]
 [StructLayout(LayoutKind.Explicit, Size = 0x210)]
 public unsafe partial struct Treasure {
+
+    /// <remarks>
+    /// Has a lot of overlap with <see cref="Flags"/>.
+    /// </remarks>
+    [FieldOffset(0x1A0)] public TreasureState State;
+
+    /// <remarks>
+    /// Starts counting down from <see cref="CountdownStartingValue_Sec"/> once spawned.  Unsure what hitting zero actually does
+    /// in the unopened state.  If the treasure contains rollable items, opening the it resets this to the claim period length
+    /// and it starts counting down again (except when lootmaster is active, in which case it resets to zero).  This value is
+    /// rounded to nearest whole number and shown as the loot window timer.
+    /// </remarks>
+    [FieldOffset(0x1A4)] public float CountdownTimer_Sec;
+
+    /// <summary>
+    /// The starting value of <see cref="CountdownTimer_Sec"/> at initial object spawn.
+    /// </summary>
+    [FieldOffset(0x1A8)] public float CountdownStartingValue_Sec;
+
+    /// <summary>
+    /// The time after opening to be able to roll on or assign drops before auto-disposition of loot.
+    /// </summary>
+    [FieldOffset(0x1AC)] public float ClaimPeriodLength_Sec;
+
+    /// <summary>
+    /// The Item sheet row IDs for the contents of this treasure that have been added to the loot window.
+    /// </summary>
+    [FieldOffset(0x1B0), FixedSizeArray] internal FixedSizeArray16<uint> _lootableItemIds; // Assuming 16 long due to the location of the count field and the layout of the loot window.  Was only able to confirm up to 12.
+    
+    /// <summary>
+    /// The number of valid item IDs in <see cref="LootableItemIds"/>.
+    /// </summary>
+    [FieldOffset(0x1F0)] public byte LootableItemCount;
+
+    [FieldOffset(0x1F4)] public int Unk_1F4; // Probably just a bool.  Seems to be something like "has been opened for looting while object exists locally for the current player", but more testing required.
+
+    /// <summary>
+    /// How long the treasure has been open.
+    /// </summary>
+    /// <remarks>
+    /// Starts from zero when opening.  Stops when fadeout begins.
+    /// </remarks>
+    [FieldOffset(0x1F8)] public float TreasureOpenTime_Sec;
+
+    /// <remarks>
+    /// Has a lot of overlap with <see cref="State"/>.
+    /// </remarks>
     [FieldOffset(0x1FC)] public TreasureFlags Flags;
 
+    /// <remarks>
+    /// Seems to be -1 for treasures that can spawn in non-fixed locations (i.e., enemies dropping coffers, deep dungeon).  Have only otherwise seen zero.
+    /// </remarks>
+    [FieldOffset(0x1FE)] public short Unk_1FE;
```

Comment:
```suggestion
    [FieldOffset(0x1FE)] private short Unk_1FE;
```
No more public unknown values will be merged.